akka-docs/src/main/paradox/persistence.md

```diff
@@ -32,6 +32,17 @@ changes to these actors from which they can rebuild internal state. This can be
 or starting from a snapshot which can dramatically reduce recovery times. Akka persistence also provides point-to-point
 communication with at-least-once message delivery semantics.
 
+@@@ note
+
+The General Data Protection Regulation (GDPR) requires that personal information must be deleted at the request of users.
```

Comment:
deleted or "forgotten"?

Reply:
I think a person has the "right to be forgotten", but saying that the information or data is forgotten is somewhat strange. Art 17 talks about "erasure of personal data" and I think deleted and erased are (close enough) synonyms.

Reply:
I just thought it would be a lead-in to the GDPR for Akka module, but no problem.